testsuite/systemtest/bintools/Compiler/tcetest_double_support.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+### TCE TESTCASE
+### title: Test double-precision operations and operands
+
+#old:mach=data/multi_addrspace.adf
+#old:src=data/multi_addrspace.c
+mach=data/doubles.adf
+src=data/doubles.cc
+program=$(mktemp tmpXXXXXX)
+
+#set -x
+tcecc -a $mach -O3 $src -o $program && \
+ttasim -a $mach -p $program --no-debugmode
+
```

Comment:
This now tests that the compilation nor simulation don't crash. It might be useful to grep the tcedisasm to ensure it actually uses the double operations and doesn't resort to software emulation or aggressively optimize the computation at compile time.